Modules/Quotas/src/Quotas.Domain/Aggregates/Tokens/Token.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Backbone.Modules.Quotas.Domain.Aggregates.Tokens;
 public class Token : ICreatedAt
 {
-    public string CreatedBy { get; set; }
+    public string? CreatedBy { get; set; }
```

Comment:
Not sure but I think this should not be nullable either - please check.

Reply:
👍